profiler/internal/stackparse/fuzz.go

```diff
@@ -0,0 +1,18 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package stackparse
+
+import "bytes"
+
+// Fuzz implements fuzzing using https://github.com/dvyukov/go-fuzz. See
+// TestFuzzCorupus for generating an initial test corpus.
+func Fuzz(data []byte) int {
+	goroutines, _ := Parse(bytes.NewReader(data))
+	if len(goroutines) > 0 {
+		return 1
+	}
+	return 0
+}
```

Comment:
I did some fuzzing but it didn't turn up any issues which hopefully means there are no crash bugs. But it could also mean that I didn't use the fuzzer correctly. The fuzzing takes very long to run, so I don't think we should make it part of the CI, but I figured we can leave this in the pkg for manual local testing.

Reply:
are there test harnesses you wrote to exercise fuzzing? We can check those in too and just mark them skipped.